src/test/scala/io/github/spark_redshift_community/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -442,6 +442,28 @@ class RedshiftSourceSuite
     mockRedshift.verifyThatExpectedQueriesWereIssued(expectedCommands)
   }
 
+  test("Include Column List adds the schema columns to the COPY query") {
+    val copyCommand =
+          "COPY \"PUBLIC\".\"test_table\" \\(\"testbyte\",\"testbool\",\"testdate\"," +
+          "\"testdouble\",\"testfloat\",\"testint\",\"testlong\",\"testshort\",\"teststring\"," +
+          "\"testtimestamp\"\\) FROM .*"
+    val expectedCommands =
+      Seq("CREATE TABLE IF NOT EXISTS \"PUBLIC\".\"test_table\" .*".r,
+          copyCommand.r)
+
+    val params = defaultParams ++ Map("include_column_list" -> "true")
+
+    val mockRedshift = new MockRedshift(
+      defaultParams("url"),
+      Map(TableName.parseFromEscaped(defaultParams("dbtable")).toString -> null))
```

Comment:
why is the schema null here? (in the Map object MAP[Table, StructType]). Should we use the TestUtils.testSchema ?

Reply:
Not sure, but I'm guessing the original author copied it from the test above this one, which for some reason uses `null`. I've changed it to `TestUtils.testSchema`.